tests/all_mode/run.sh

```diff
@@ -159,6 +159,16 @@ function run() {
     check_log_not_contains $WORK_DIR/worker1/log/dm-worker.log "123456"
     check_log_not_contains $WORK_DIR/worker2/log/dm-worker.log "123456"
 
+    # test drop table if exists
+    run_sql_source1 "drop table if exists \`all_mode\`.\`tb1\`;"
+    run_sql_source1 "drop table if exists \`all_mode\`.\`tb1\`;"
+    run_sql_source2 "drop table if exists \`all_mode\`.\`tb2\`;"
+    run_sql_source2 "drop table if exists \`all_mode\`.\`tb2\`;"
+    sleep 2
```

Comment:
how about retry and check downstream table dropped, maybe quicker than 2s

Reply:
I want to check no error exist.

Reply:
just want to replace sleep 2s this line 🤣

Reply:
check log in 5bb5f63